From d9d8e55c8f2e291a9b936068b25478f31f0997b4 Mon Sep 17 00:00:00 2001
From: "flinux@vongellhorn.ch" <flinux@vongellhorn.ch>
Date: Fri, 18 Oct 2019 14:28:59 +0200
Subject: [PATCH 2/2] Use explicit path of the gtk-module

Despite it working just about everywhere else (including installing it in the right place), it seems to break here.
---
 gtk3proxies/preview3.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gtk3proxies/preview3.c b/gtk3proxies/preview3.c
index cb3a985..f2873b2 100644
--- a/gtk3proxies/preview3.c
+++ b/gtk3proxies/preview3.c
@@ -68,7 +68,7 @@ int main(int argc, char **argv)
             sscanf(argv[1], "%ld", &wid);
     }
     
-    const char* ui_file = DATA_DIR "/preview.ui";
+    const char* ui_file = "/usr/share/kcm-gtk-module/preview.ui";
  
     GtkBuilder *builder = gtk_builder_new();
     if( ! gtk_builder_add_from_file( builder, ui_file, &error ) ) {
-- 
2.23.0

